src/Features/Core/Portable/DesignerAttributes/IRemoteDesignerAttributeService.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.CodeAnalysis.DesignerAttributes
 {
     internal interface IRemoteDesignerAttributeService
     {
-        Task<DesignerAttributeResult> ScanDesignerAttributesAsync(DocumentId documentId);
+        Task<DesignerAttributeResult> ScanDesignerAttributesAsync(DocumentId documentId, CancellationToken cancellationToken);
```

Comment:
:memo: For callers, this really doesn't change much. The JSON RPC library we use for remote invocations provides the token automatically if the method signature requires it. If people are using IVT to *implement* these interfaces, it would be a breaking change.